assets/css/v2/pre_fare/evergreen/image.scss

```diff
@@ -0,0 +1,57 @@
+.evergreen-content-image__container {
+  background-color: #e4e6e1;
+  overflow: hidden;
+}
+
+.flex-two-medium__left,
+.flex-two-medium__right {
+  .evergreen-content-image__container {
+    border-radius: 16px;
+    box-shadow: 0px 12px 24px 0px rgba(23, 31, 38, 0.25);
+  }
+
+  .evergreen-content-image__container,
+  .evergreen-content-image__image {
+    width: 480px;
+    height: 580px;
+  }
+}
+
+.flex-one-large__large {
+  .evergreen-content-image__container {
+    border-radius: 16px;
+    box-shadow: 0px 12px 24px 0px rgba(23, 31, 38, 0.25);
+  }
+
+  .evergreen-content-image__container,
+  .evergreen-content-image__image {
+    width: 1024px;
+    height: 576px;
+  }
+}
+
```

Comment:
I think this is missing styles for the `:lower_right` slot—corresponding class "body-right-normal__lower". Otherwise looks good!